CONTRIBUTING.md

```diff
@@ -35,10 +47,25 @@ All of the automated tests for this project need to pass before your submission
 
 Example of a good commit message:
 
-```
+```plain
 Improve contributing guidelines. Fixes #10
 
-Improve contributing docs and consolidate them in the standard location https://help.github.com/articles/setting-guidelines-for-repository-contributors/
+Improve contributing docs and consolidate them in the standard location
+https://help.github.com/articles/setting-guidelines-for-repository-contributors/
 ```
 
-### For more information on Cumulus governance, see the [Cumulus Code Contribution Guidelines](https://docs.google.com/document/d/14J_DS6nyQ32BpeVjdR-YKfzHAzFB299tKghPGshXUTU/edit) and [the Cumulus Wiki](https://wiki.earthdata.nasa.gov/display/CUMULUS/Cumulus).
+**For more information on Cumulus governance, see the [Cumulus Code Contribution
+Guidelines] and the [Cumulus Wiki]**
+
+[search open issues]:
+  https://github.com/cumulus-nasa/cumulus-message-adapter-python/issues
+[create a new issue]:
+  https://github.com/cumulus-nasa/cumulus-message-adapter-python/issues/new
```

Comment:
Please also update this file with new URL.
https://github.com/nasa/cumulus-message-adapter-python/blob/master/example/requirements.txt